src/content/docs/specs/episodes/get-all.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Get all episodes
+description: Get all episodes for a user
+sidebar:
+  order: 3
+  badge:
+    text: Core
+    variant: caution
+---
+
+import CoreAction from "@partials/_core-action.mdx";
+
+<CoreAction />
+
+```http title="Endpoint"
+GET /v1/episodes
+```
+
+This endpoint enables clients to return all episode information relating to the authenticated user. It returns pagination information and an array of `episodes`.
+
+While supported, this endpoint is expected to be used rarely in practice. More often, episodes are retrieved per queue (TBD) or per subscription (TBD).
```

Comment:
Did we have an idea yet where/how episodes of a given subscription should be retrieved? `/v1/episodes?subscription=x` or `GET /v1/subscriptions/{guid}/episodes`?

Side note: I guess this episode will be used a lot actually, just with the `since` parameter most of the time.